src/components/Login.js

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable no-undef */
+import {initializeApp} from "firebase/app";
+import React, {useState} from "react";
+import {useNavigate} from "react-router-dom";
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import {Button} from "../styles/Profile.module";
+import {getFirestore, doc, setDoc, onSnapshot} from "firebase/firestore";
+import {LoginWrapper} from "../styles/Login.module";
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
+  authDomain: process.env.REACT_APP_FIREBASE_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECTID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGEBUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGINGSENDERID,
+  appId: process.env.REACT_APP_FIREBASE_APPID,
+  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
+};
+
+// Initialize Firebase
+const app = initializeApp(firebaseConfig);
+const auth = getAuth();
+const db = getFirestore(app);
```

Comment:
DRY. You could consider managing the firebase instance into redux or initialize only once in App.js.

Reply:
Initialize firebaseConfig  in App.js and pass it to components as props.